### PR TITLE
Team: Add Najam

### PR DESCRIPTION
**Pull request by psiinon**
Screenshot 2022-10-10 at 17-09-14 OWASP ZAP – ZAP Team

Screenshot 2022-10-10 at 17-09-23 OWASP ZAP – Team – Najam Ul Saqib

Signed-off-by: Simon Bennetts <psiinon@gmail.com>

### CODE REVIEW

**Review by kingthorin — APPROVED** [review]

File: site/content/docs/team/najam.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+---
+handle: najam
```
Comment (njmulsqb) [inline site/content/docs/team/najam.md:2]:
```suggestion
handle: njmulsqb
```
Please update the handle

File: site/content/docs/team/najam.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+#### Talks & Videos
+
+Najam runs a [YouTube channel](https://youtube.com/channel/UC1igugJ0bSePPDqIWOje2ZA) in which he presents walkthroughs for vulnerable machines in his mother language i.e Urdu. 
+
+
+#### Other Work
+
+1. Works full time as an Application Security Engineer
+1. Hunts for security issues in free time, have found issues in different companies.
```
Comment (njmulsqb) [inline site/content/docs/team/najam.md:27]:
```suggestion
2. Hunts for security issues in free time, have found security bugs in different companies.
```
Minor tweaks

**Comment by psiinon** [comment]
Made those changes - looking ok now @njmulsqb ?
btw I didnt change the "1." to "2." because thats the MD syntax for a numbered list, if you look at the original screenshot you'll see it correct 😉

**Comment by njmulsqb** [comment]
Oh Okay. Yeah looks good now! @psiinon

**Review by thc202 — APPROVED** [review]

**Pull request merged**